spec/RestQuery.spec.js

```diff
@@ -419,4 +419,28 @@ describe('RestQuery.each', () => {
     expect(resultsOne.length).toBe(1);
     expect(resultsTwo.length).toBe(1);
   });
+
+  it('test afterSave response object is return', done => {
+    Parse.Cloud.afterSave('TestObject2', function(req) {
+      const jsonObject = req.object.toJSON();
```

Comment:
I think it is not so straightforward for others users to use this capability. Can you test just using the code below?

```
req.object.unset('todelete');
req.object.set('toadd', true);
return req.object;
```

Reply:
It's actually an other issue I had. If you do that you will have : 
```javascript
{
  "tokeep": true,
  "createdAt": "2019-07-18T09:12:58.496Z",
  "todelete": {
    "__op": "Delete"
  },
  "updatedAt": "2019-07-18T09:12:58.496Z",
  "toadd": true,
  "objectId": "zC2AmMvGxM"
}
``` 

the `__op` is not really great response :/

Reply:
Can you try like this?

```
req.object.unset('todelete');
req.object.set('toadd', true);
return req.object._toFullJSON();
```

Reply:
With that I have :
```javascript
{ todelete: { __op: 'Delete' },
     tokeep: true,
     createdAt: '2019-07-25T13:49:32.552Z',
     updatedAt: '2019-07-25T13:49:32.552Z',
     toadd: true,
     objectId: '4lgUT2xWo5',
     __type: 'Object',
     className: 'TestObject2' }
```

with `toJson` :

```javascript
{ todelete: { __op: 'Delete' },
     tokeep: true,
     createdAt: '2019-07-25T13:49:32.552Z',
     updatedAt: '2019-07-25T13:49:32.552Z',
     toadd: true,
     objectId: '4lgUT2xWo5' }
```